go/vt/topo/cluster_event.go

```diff
@@ -0,0 +1,143 @@
+package topo
+
+import (
+	"context"
+	"sort"
+
+	"google.golang.org/protobuf/proto"
+	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+	"vitess.io/vitess/go/vt/vterrors"
+)
+
+type WatchTopoEventData struct {
```

Comment:
A top-level comment on the struct will be good. What it encapsulates + how it is intended to be used.
(I know most source files in vitess don't currently have this, but we have to start somewhere).